### PR TITLE
Add custom messages

### PR DESCRIPTION
**Pull request by VerasThiago**
#### What is the purpose of this pull request?
Custom messages of `update` `deprecated` and `outdated`

#### What problem is this solving?
Show personalised message based on user OS

#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [X] Update `CHANGELOG.md`

### CODE REVIEW

File: src/CLIPreTasks/DeprecationChecker/DeprecationChecker.ts [hunk 1]
```diff
@@ -24,9 +24,11 @@ export class DeprecationChecker {
       return
     }
 
-    const errMsg = `${chalk.bold(
-      `Your Toolbelt version (${pkgJson.version}) was deprecated`
-    )}. ${Messages.UPDATE_TOOLBELT()}`
+    const errMsg = [
+      `${chalk.bold(`Your Toolbelt version (${pkgJson.version}) was deprecated`)}.`,
+      `To deprecate, you must use the same method you used to install. As the following example(s):`,
+      ...deprecateMessageSwitch(),
```
Comment (dhasuda) [inline src/CLIPreTasks/DeprecationChecker/DeprecationChecker.ts:30]: Is this a message for the toolbelt user, right? Why would they need to deprecate the toolbelt? Or did I misunderstand it?

Wouldn't it be better
```
To update your toolbelt, you must use the same method you used to install. (...)
```
?

File: src/lib/constants/Messages.ts [hunk 2]
```diff
@@ -22,4 +23,90 @@ export const Messages = {
...
+      `curl https://vtex-toolbelt-test.s3.us-east-2.amazonaws.com/uninstall.sh | sh`
+    )}
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `curl https://vtex-toolbelt-test.s3.us-east-2.amazonaws.com/install.sh | sh`
+    )}`,
+  DEPRECATE_TOOLBELT_CHOCOLATEY: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(`chocolatey`)}, deprecate running:
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(`choco uninstall vtex`)}.
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(`choco install vtex`)}.`,
+}
```
Comment (dhasuda) [inline src/lib/constants/Messages.ts:58]: Same here. What does "deprecate" mean? Is it update or is it deprecate the toolbelt?

**Review by dhasuda — COMMENTED** [review]
I'm a little confused about what "deprecate" means in the messages

File: src/CLIPreTasks/DeprecationChecker/DeprecationChecker.ts (hunk 1 above)
Comment (VerasThiago) [reply src/CLIPreTasks/DeprecationChecker/DeprecationChecker.ts:30]: Yeah, you are right! I will fix it

File: src/lib/constants/Messages.ts (hunk 2 above)
Comment (VerasThiago) [reply src/lib/constants/Messages.ts:58]: Same here

File: src/CLIPreTasks/DeprecationChecker/DeprecationChecker.ts [hunk 3]
```diff
@@ -24,9 +24,11 @@ export class DeprecationChecker {
       return
     }
 
-    const errMsg = `${chalk.bold(
-      `Your Toolbelt version (${pkgJson.version}) was deprecated`
-    )}. ${Messages.UPDATE_TOOLBELT()}`
+    const errMsg = [
+      `${chalk.bold(`Your Toolbelt version (${pkgJson.version}) was deprecated`)}.`,
+      `To update, you must use the same method you used to install. As the following example(s):`,
+      ...deprecateMessageSwitch(),
```
Comment (dhasuda) [inline src/CLIPreTasks/DeprecationChecker/DeprecationChecker.ts:30]: Can you change from `deprecatedMessageSwitch` to `updateFromDeprecatedMessageSwitch` or something like this?

File: src/lib/constants/Messages.ts [hunk 4]
```diff
@@ -22,4 +23,90 @@ export const Messages = {
...
+      `curl https://vtex-toolbelt-test.s3.us-east-2.amazonaws.com/uninstall.sh | sh`
+    )}
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `curl https://vtex-toolbelt-test.s3.us-east-2.amazonaws.com/install.sh | sh`
+    )}`,
+  DEPRECATE_TOOLBELT_CHOCOLATEY: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(`chocolatey`)}, update running:
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(`choco uninstall vtex`)}.
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(`choco install vtex`)}.`,
+}
```
Comment (dhasuda) [inline src/lib/constants/Messages.ts:58]: I suggest changing from `DEPRECATE_` to `UPDATE_FROM_DEPRECATED` or something that makes more explicit that it's a update instruction, not a deprecate instruction

File: src/lib/constants/Messages.ts [hunk 5]
```diff
@@ -22,4 +23,90 @@ export const Messages = {
...
+        break
+      default:
+        break
+    }
+  }
+
+  return allMessages
+}
+
+export function deprecateMessageSwitch() {
```
Comment (dhasuda) [inline src/lib/constants/Messages.ts:87]: Same here about deprecate

**Review by dhasuda — CHANGES_REQUESTED** [review]
Few requests about `deprecation`

**Review by dhasuda — APPROVED** [review]
LGTM!!

**Pull request merged**